packages/rocketchat-i18n/i18n/en.i18n.json

```diff
@@ -2780,13 +2780,14 @@
   "UTF8_Names_Validation": "UTF8 Names Validation",
   "UTF8_Names_Validation_Description": "RegExp that will be used to validate usernames and channel names",
   "Validate_email_address": "Validate Email Address",
+  "Verification_email_body": "You have succesfully created an account on [Site_Name]. Please, click on the button bellow to confirm your email address and finish your registration",
```

Comment:
bellow -> below

Reply:
> bellow -> below

🤦‍♂️  I was so confident it was `bellow`

Reply:
😁 to Bellow is to yell or roar but in a deep voice